draft-ietf-quic-transport.md

```diff
@@ -3136,12 +3136,10 @@ This recommendation is in keeping with standard practice for TCP {{?RFC5681}}.
 
 In order to assist loss detection at the sender, an endpoint SHOULD send an ACK
 frame immediately on receiving an ack-eliciting packet that is out of order. The
```

Comment:
We still need to know what "out of order" means.  I presume that this is "when an ack-eliciting packet arrives with a packet number that more than one greater than any previously received packet" or something like that.

Reply:
Agreed. I think that's the definition we are looking for to close #3347, though I am now beginning to wonder if we need "ack-eliciting" within the condition.

Shouldn't receiving a packet with a PN gap, regardless of the packet being ACK-eliciting or not, trigger an immediate ACK?

Reply:
Yeah, I should have moved the definition from the recovery draft when the ACK text was moved.

I believe moving text is an editorial change, so how about I do that in a separate PR?

Reply:
#3369

Reply:
@martinthomson Not sure about "when an ack-eliciting packet arrives with a packet number that more than one greater than **any** previously received packet". If a delayed packet has been received, would not you want to notify the other endpoint ASAP, in case it did not retransmit the data, yet?

I'd think "out of order" means "_any packet with a packet number other than one greater then the largest previously received packet number_".  (Or, optimizing a bit, "_any ack-eliciting packet with a packet number other than one greater then the largest previously received packet number, or a non-ack-eliciting packet with a packet number more than one greater then the largest previously received packet number_".)